### PR TITLE
Bug fix: Fixed star width on Firefox

### PR DESCRIPTION
**Pull request by slannigan**
On Firefox, setting the `<svg width='xx' height='xx'>` is not sufficient. If only this is done, then the svgs will attempt to take up the full width of the container div.

Fixed the problem by additionally setting the width and height as part of the `style` attribute, so it looks like this: `<svg width='xx' height='xx' style='width: xx; height: xx;'>`.

Before the fix: https://puu.sh/vQIVT/011147950b.png
After the fix: https://puu.sh/vQIWA/72f87e85e5.png

### CODE REVIEW

File: addon/components/star-rating.js [hunk 1]
```diff
@@ -20,6 +20,17 @@ const RatingComponent = Component.extend({
   svgPath: 'M25.326,10.137c-0.117-0.361-0.431-0.625-0.807-0.68l-7.34-1.066l-3.283-6.651 c-0.337-0.683-1.456-0.683-1.793,0L8.82,8.391L1.48,9.457c-0.376,0.055-0.689,0.318-0.807,0.68c-0.117,0.363-0.02,0.76,0.253,1.025 l5.312,5.178l-1.254,7.31c-0.064,0.375,0.09,0.755,0.397,0.978c0.309,0.225,0.717,0.254,1.054,0.076L13,21.252l6.564,3.451 c0.146,0.077,0.307,0.115,0.466,0.115c0.207,0,0.413-0.064,0.588-0.191c0.308-0.223,0.462-0.603,0.397-0.978l-1.254-7.31 l5.312-5.178C25.346,10.896,25.443,10.5,25.326,10.137z',
   viewBox: '0 0 26 26',
 
+  style: Ember.computed('width', 'height', function() {
+    var style = '';
+    if (this.get('width')) {
```
Comment (trmcnvn) [inline addon/components/star-rating.js]: Could you change this and other instances to use the `get(this, 'width')` format to match the rest of the code

File: addon/components/star-rating.js [hunk 2]
```diff
@@ -20,6 +20,17 @@ const RatingComponent = Component.extend({
   svgPath: 'M25.326,10.137c-0.117-0.361-0.431-0.625-0.807-0.68l-7.34-1.066l-3.283-6.651 c-0.337-0.683-1.456-0.683-1.793,0L8.82,8.391L1.48,9.457c-0.376,0.055-0.689,0.318-0.807,0.68c-0.117,0.363-0.02,0.76,0.253,1.025 l5.312,5.178l-1.254,7.31c-0.064,0.375,0.09,0.755,0.397,0.978c0.309,0.225,0.717,0.254,1.054,0.076L13,21.252l6.564,3.451 c0.146,0.077,0.307,0.115,0.466,0.115c0.207,0,0.413-0.064,0.588-0.191c0.308-0.223,0.462-0.603,0.397-0.978l-1.254-7.31 l5.312-5.178C25.346,10.896,25.443,10.5,25.326,10.137z',
   viewBox: '0 0 26 26',
 
+  style: Ember.computed('width', 'height', function() {
```
Comment (trmcnvn) [inline addon/components/star-rating.js]: Change this to the shim style approach:

```javascript
import computed from 'ember-computed';

style: computed('width', 'height', function() { ... }),
```

File: addon/components/star-rating.js [hunk 3]
```diff
@@ -20,6 +20,17 @@ const RatingComponent = Component.extend({
...
+    var style = '';
+    if (this.get('width')) {
+      style += `width: ${this.get('width')}; `;
+    }
+    if (this.get('height')) {
+      style += `height: ${this.get('height')};`;
+    }
+    return style;
```
Comment (trmcnvn) [inline addon/components/star-rating.js]: This'll need to be marked as HTML Safe.

```javascript
import { htmlSafe } from 'ember-string';

return htmlSafe(style);
```

**Review by trmcnvn — CHANGES_REQUESTED** [review]
Thanks! I wasn't aware of this.

Just a few changes and will be good to merge :+1:

**Comment by slannigan** [comment]
Whoops, apologies, just completely ignored your existing patterns. Fixed!

**Comment by danconnell** [comment]
:+1:

**Pull request merged**